auslib/web/admin/emergency_shutoff.py

```diff
@@ -0,0 +1,154 @@
+import time
```

Comment:
Thank you for implementing these directly and not adding more indirection with mapper.py!